sdk-src/src/main/java/com/hpe/adm/nga/sdk/network/google/GoogleHttpClient.java

```diff
@@ -58,6 +60,8 @@
     protected String octaneUserValue;
     protected final String urlDomain;
     protected Authentication lastUsedAuthentication;
+    private final Map<OctaneHttpRequest, OctaneHttpResponse> cachedRequestToResponse = new HashMap<>();
```

Comment:
That's the point of caching.  Etag Cache means that the result is saved on the client.  Once the process finishes it will be gone (which is different from browsers since they save even when closing the session)